binderhub/launcher.py

```diff
@@ -174,14 +192,9 @@ def username_from_repo(self, repo_url):
             else:
                 body = ''
 
-            app_log.error("Error starting server for %s: %s\n%s",
-                username, e, body,
-            )
+            app_log.error("Error starting server{} for user {}: {}\n{}".
```

Comment:
missing space between `server{}`

Reply:
`_server_name` already contains the space